src/chrome/content/code/ApplicableList.js

```diff
@@ -212,13 +212,9 @@ ApplicableList.prototype = {
 
   add_command: function(rule) {
       var command = this.document.createElement("command");
-      if (typeof rule.id !== 'number') {
-        this.log(WARN, "Rule has invalid id " + rule.id);
-        return;
-      }
       command.setAttribute('id', rule.id+'-command');
```

Comment:
I suspect that they would ding you for this too. You may want to json stringify this and the next line as well.